### ISSUE TITLE
Very nice

### ISSUE THREAD

**Issue by dennisTGC** [stepanzubkov/where-is-my-sddm-theme#2]
Hi, I found it very nice. However, when I typed my password wrong a white screen appeared and it scared me thinking something was wrong with the theme and I couldn't login anymore.. lol. 

But 1 thing is holding me back to make it default, that is that there are other family members using this system, so I would like to have an option to select another user. 

Also, the touchscreen keyboard is appearing, I do have a touchpad but not a touchscreen.

**Comment by stepanzubkov** [comment]
Hi! I tested this theme again, but I could not reproduce bug with white screen (This is bug, not theme feature).

I was thinking about selection another user and another session. I will not add graphical selection box, because it contradicts theme idea. But my idea is add cyclic selection of user by pressing one of keyboard key (`<F2>` for example). After pressing this key, selected username will appear on the screen above password input.

You can disable virtual keyboard by setting line `InputMethod=qtvirtualkeyboard` to `InputMethod=` in sddm config file. SDDM config is located in `/etc/sddm.conf` or `/etc/sddm.conf.d/kde_settings.conf`

Thanks for testing the theme and reporting the bug! 😃

**Comment by dennisTGC** [comment]
Hi, 

I have a Alienware R14 and 2x Dell 27" 4k screens, am on kubuntu 23.04. Let me know if I need to do some more testing. I found out that the first time logon is hindered by the virtualkeyboard and defaults to my right screen, somehow resulting in the first character being filled. I will turn off the virtualkeyboard and test some more. I am on my laptop for now, so i'll test it later.

I moved to your other theme zuru but let me know if I need some testing. The F2 would be an acceptable solution, but without the selection option it's just not workable for my situation.
